### PR TITLE
feat: support mocker config in the command line

### PR DESCRIPTION
**Pull request by vanpipy**
For #144

### CODE REVIEW

File: packages/core/package.json [hunk 1]
```diff
@@ -47,5 +47,8 @@
     "http-proxy": "1.18.1",
     "local-ip-url": "1.0.3",
     "path-to-regexp": "6.2.0"
+  },
+  "devDependencies": {
+    "minimist": "^1.2.5"
```
Comment (jaywcjlove) [inline packages/core/package.json:52]: @vanpipy 

```diff
- "devDependencies": {
-    "minimist": "^1.2.5"
- }
+ "dependencies": {
+    "minimist": "^1.2.5"
      ....
+ }
```

Comment (vanpipy) [reply packages/core/package.json]: Remove the `devDependencies` and move the `miniimst` into `dependencies` already.

**Comment by vanpipy** [comment]
Here, I have a question is that the unit test. Although I run the code build and debug through the use case follow the document, I cannot ensure the code changed runs is the same as the original code. Can I add some basic some unit tests in the project for the current file?

**Comment by jaywcjlove** [comment]
@vanpipy sure, welcome to add unit tests.

**Pull request merged**